cmd/packaging/packaging.go

```diff
@@ -200,25 +217,18 @@ func (t *packagingTask) Pack(buildVersion string) {
 		os.Exit(1)
 	}
 
-	packagingScript := executeStringTemplate(t.packagingScriptTemplate, t.getTemplateData(projectName, buildVersion))
-	runPackaging(tmpPath, packagingScript)
-	var outputFileName string
-	if t.outputFileUsesApplicationName {
-		outputFileName += config.GetConfig().GetApplicationName(projectName)
-	} else {
-		outputFileName += config.GetConfig().GetPackageName(projectName)
-	}
-	if t.outputFileContainsVersion {
-		if t.outputFileUsesApplicationName {
-			outputFileName += " "
-		} else {
-			outputFileName += "-"
-		}
-		outputFileName += buildVersion
+	relativeOutputFilePath, err := t.packagingFunction(tmpPath, applicationName, packageName, executableName, version, release)
+	if err != nil {
+		log.Errorf("%v", err)
+		log.Warnf("Packaging is very experimental and has only been tested on Linux.")
+		log.Infof("To help us debuging this error, please zip the content of:\n       \"%s\"\n       %s",
+			log.Au().Blue(tmpPath),
+			log.Au().Green("and try to package on another OS. You can also share this zip with the go-flutter team."))
```

Comment:
This sentence is a bit weird....

In general I'm wondering if we should directly ask the user to zip their (possibly proprietary) source code and send it to us. I'd rather have they open an issue, and then we can see if we can work towards a reduced project that replicates the issue.

Reply:
I would also prefer something more precise

Reply:
Would you add something to the update notice or change it in any way?

Reply:
In this PR? Which line do you mean?